packages/salesforcedx-vscode-apex-replay-debugger/src/telemetry/telemetry.ts

```diff
@@ -31,9 +30,10 @@ export class TelemetryService {
     this.reporters = reporters;
   }
 
-  public sendExtensionActivationEvent(hrstart: [number, number]): void {
+  public sendExtensionActivationEvent(hrstart?: number): void {
+    const startTime = hrstart || globalThis.performance.now();
```

Comment:
I've also noticed this is duplicate code. I am thinking of moving the logic to our utils package.